.circleci/config.yml

```diff
@@ -419,6 +419,72 @@ commands:
       - store_artifacts:
           path: /tmp/core_dumps
 
+  # A job that runs the fog-local-network script, building things in release mode
+  # This is the only test in the system that tests that change outputs are spendable.
+  # The test ensures this because fog distro is set to only give one tx out per account,
+  # so the test client eventually is forced to spend change.
+  run-fog-local-network-tests-release:
+    steps:
+      - run:
+          name: fog_local_network
+          command: |
+            # tell the operating system to remove the file size limit on core dump files
+            ulimit -c unlimited
+            openssl genrsa -out Enclave_private.pem -3 3072
+            export SGX_MODE=SW
+            export IAS_MODE=DEV
+            export CONSENSUS_ENCLAVE_PRIVKEY=$(pwd)/Enclave_private.pem
+            export INGEST_ENCLAVE_PRIVKEY=$(pwd)/Enclave_private.pem
+            export LEDGER_ENCLAVE_PRIVKEY=$(pwd)/Enclave_private.pem
+            export VIEW_ENCLAVE_PRIVKEY=$(pwd)/Enclave_private.pem
+            export MC_LOG=debug
+
+            cargo build -p mc-util-keyfile -p mc-util-generate-sample-ledger -p mc-consensus-service -p mc-ledger-distribution -p mc-admin-http-gateway -p mc-util-grpc-admin-tool -p mc-mobilecoind -p mc-crypto-x509-test-vectors -p mc-fog-distribution -p mc-fog-test-client -p mc-fog-ingest-server -p mc-fog-ingest-client -p mc-fog-view-server -p mc-fog-report-server -p mc-fog-ledger-server -p mc-fog-sql-recovery-db --release
```

Comment:
Optional/Equivalent: One line per package, and sort them.

Reply:
Ack